x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceDetailTabs.tsx

```diff
@@ -56,16 +72,37 @@ export function ServiceDetailTabs({ urlParams }: Props) {
   const tabs = [transactionsTab, errorsTab];
   if (agentName && !isRumAgentName(agentName)) {
     const metricsTab = {
-      title: i18n.translate('xpack.apm.serviceDetails.metricsTabLabel', {
-        defaultMessage: 'Metrics'
-      }),
-      path: `/services/${serviceName}/metrics`,
+      link: (
+        <MetricOverviewLink serviceName={serviceName}>
+          {i18n.translate('xpack.apm.serviceDetails.metricsTabLabel', {
+            defaultMessage: 'Metrics'
+          })}
+        </MetricOverviewLink>
+      ),
       render: () => <ServiceMetrics agentName={agentName} />,
       name: 'metrics'
     };
 
     tabs.push(metricsTab);
   }
 
-  return <HistoryTabs tabs={tabs} />;
+  const selectedTab = tabs.find(serviceTab => serviceTab.name === tab);
+
+  return (
+    <>
+      <EuiTabs>
+        {tabs.map(serviceTab => (
+          <EuiTabLink
+            onClick={() => null}
```

Comment:
is `onClick` required?

Reply:
Yes. Unfortunately 🙁. I've checked in #eui if this was the recommened approach, and it is.

Reply:
Fair enough. 
Still wondering why EUI doesn't set it by default... or just don't require it.